src/fondue/Fondue.js

```diff
@@ -760,7 +777,45 @@ export default class Fondue {
 				});
 			}
 
-			return currentAllGlyphs;
+			// Chained context substitution
+			if (lookup.lookupType === 6) {
```

Comment:
Just a thought: It might be nice to split this big function up in several small functions, one per lookup type. That would make it a lot easier to find the implementation for a specific lookup type, instead of having to scroll through this large function to find the right section. WDYT?

Reply:
Maybe even export this to a separate file, as this is really going to be the biggest chunk of logic in the engine. The rest is basically massaging data to be easier consumable by the site, but this is a lot more complex. is there a structure I could follow here, like a best practise?

Reply:
Indeed. What do you think about something like this?

```js
parseLookup(lookup) {
  const lookupParsers = {
    1: require("./lookups/1.js"),
    2: require("./lookups/1.js"),
    ...
  };
  
  return lookupParsers[lookup.lookupType](lookup);
}
```

Where every file in `lookups/` would export a function, that takes the lookup (or whatever arguments are needed).
I hope you can come up with better names than `1.js`, but you get the point :-)

Reply:
Thanks! That sounds really good. I think I can come up with better names :-)

To not further delay this particular PR, I've moved this to an issue so we can keep track of it: https://github.com/Wakamai-Fondue/wakamai-fondue-engine/issues/44